CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
-### 2.3.2 (Next)
+### 3.0.0 (Next)
```

Comment:
We also need to increment the version in https://github.com/acm19/aws-request-signing-apache-interceptor/blob/756a6e3b95ea1c077fcfa7d4fe8c4fa143e93bd3/pom.xml#L7.